security/pkg/nodeagent/sds/monitoring.go

```diff
@@ -111,6 +111,15 @@ var (
 			Subsystem: "sds_service",
 			Help:      "The date after which a pushed server certificate expires. Expressed as a Unix Epoch Time.",
 		}, []string{"resourcePerConn"})
+
+	// totalSecretUpdateFailureCounts records total number of secret update failures reported by
+	// proxy in SDS request <error_detail> field.
+	totalSecretUpdateFailureCounts = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: "citadel_agent",
+		Subsystem: "sds_service",
+		Name:      "total_secret_update_failures",
```

Comment:
I think promethues may already append `_total` to the end of the metric name if its a counter? Either that or its a common pattern to make it `something_total` not `total_something` I think